packages/esm-patient-attachments-app/src/attachments/camera-upload.component.tsx

```diff
@@ -75,7 +75,13 @@ const CameraUpload: React.FC<CameraUploadProps> = ({ onSavePhoto, onTakePhoto, c
               <label htmlFor="uploadPhoto" className={styles.choosePhoto}>
                 {t('selectPhoto', 'Select local photo instead')}
               </label>
-              <input type="file" id="uploadPhoto" accept="image/*" className={styles.uploadFile} onChange={upload} />
+              <input
+                type="file"
+                id="uploadPhoto"
+                accept="image/*, application/pdf"
+                className={styles.uploadFile}
+                onChange={upload}
+              />
```

Comment:
@jnsereko thanks for the review and comment. What if I use `id="uploadphotoOrPdf" won't it solve the issue? why I ask so is because Doc is relative it can mean a excel, word e.t.c. Its just a stupid thinking and I stand to be corrected.

Reply:
i was just giving an example but i guess `id="uploadphotoOrPdf" is better and more relevant to the issue

Reply:
@jnsereko Thanks so much!